tardis/model/base.py

```diff
@@ -189,6 +196,33 @@ def abundance(self):
         abundance.columns = range(len(abundance.columns))
         return abundance
 
+    def decay(self, time_explosion, normalize=True):
+
+        if not self.decayed:
+            self.decayed = True
+        else:
+            raise ValueError("Isotopic Abundance already decayed")
+
+        isotope_abundance = self.raw_isotope_abundance.decay(time_explosion)
+
+        #Set atomic_number as index in isotopic_abundance dataframe
+        isotope_abundance = isotope_abundance.reset_index().drop(
+            'mass_number', axis=1).set_index(['atomic_number'])
+
+        #Merge abundance dataframes
+        modified_df = isotope_abundance.append(self.abundance)
```

Comment:
you definitely do not want to append - but add the decayed dataframe to the abundances.